api/storage_service.cc

```diff
@@ -455,6 +455,14 @@ static future<json::json_return_type> describe_ring_as_json(sharded<service::sto
     co_return json::json_return_type(stream_range_as_array(co_await ss.local().describe_ring(keyspace), token_range_endpoints_to_json));
 }
 
+std::vector<table_id> get_table_ids(const std::vector<table_info>& table_infos) {
+    std::vector<table_id> table_ids{table_infos.size()};
+    boost::transform(table_infos, table_ids.begin(), [] (const auto& ti) {
+        return ti.id;
+    });
+    return table_ids;
+}
```

Comment:
Doesn't this conflict with https://github.com/scylladb/scylladb/pull/12714?

Reply:
Actually, this commit is cherry-picked from https://github.com/scylladb/scylladb/pull/12714. There will be merge conflicts between the two anyway, so I think dropping the commit from the second to be queued does not hurt much.